Tests/EventListener/RequestResponseListenerTest.php

```diff
@@ -683,7 +683,7 @@ private function createRouteCollecitonWithRouteAndRoutingOption($returnValue, $t
     {
         $route = $this->getMockBuilder('Symfony\Component\Routing\Route')->disableOriginalConstructor()->getMock();
         $route->expects($this->exactly($times))->method('getOption')->will($this->returnValue($returnValue));
-        $routeCollection = $this->getMock('Symfony\Component\Routing\RouteCollection');
+        $routeCollection = $this->createMock('Symfony\Component\Routing\RouteCollection');
```

Comment:
`getMock()` is deprecated: https://travis-ci.org/suncat2000/MobileDetectBundle/jobs/275668490#L539